internal/app/go-filecoin/paymentchannel/testing/fake_mgr_api.go

```diff
@@ -100,50 +101,49 @@ func (f *FakePaymentChannelAPI) Send(_ context.Context,
 
 // testing methods
 
-// StubCreatePaychActorMessage sets up a message response, with desired exit code and block height
-func (f *FakePaymentChannelAPI) StubCreatePaychActorMessage(
+// GenCreatePaychActorMessage sets up a message response, with desired exit code and block height
+func GenCreatePaychActorMessage(
 	t *testing.T,
 	clientAccountAddr, minerAccountAddr, paychUniqueAddr address.Address,
 	amt abi.TokenAmount,
 	code exitcode.ExitCode,
-	height uint64) {
+	height uint64) (cid.Cid, MsgResult) {
 
 	newcid := shared_testutil.GenerateCids(1)[0]
 
 	msg := types.NewUnsignedMessage(clientAccountAddr, builtin.InitActorAddr, 1,
 		types.NewAttoFIL(amt.Int), builtin.MethodsInit.Exec, []byte{})
-	msg.GasPrice = defaultGasPrice
-	msg.GasLimit = defaultGasLimit
+	msg.GasPrice = types.NewAttoFILFromFIL(100)
+	msg.GasLimit = gas.NewGas(300)
```

Comment:
Not sure if it matters here, but 300 is too low, gas needs to be at least 5000 to work.